File: Tournament/Models/ViewModels/TableRow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Tournament.Models.ViewModels
{
    public class TableRow
    {
        public Team Team { get; set; }
        public int Played { get; set; }
        public int Wins { get; set; }
        public int Draws { get; set; }
        public int Losses { get; set; }
        public int GoalsScored { get; set; }
        public int GoalsConceded { get; set; }
        public int Points { get; set; }
    }
}

File: frontend/tournament/src/components/group-card/table-row.js
import React, { Component } from 'react';

class TableRow extends Component {
  render() {
    return (
      <tr>
        <td>{this.props.name}</td>
        <td>{this.props.played}</td>
        <td>{this.props.wins}</td>
        <td>{this.props.draws}</td>
        <td>{this.props.losses}</td>
        <td>{this.props.goalsScored}-{this.props.goalsConceded}</td>
        <td>{this.props.points}</td>
      </tr>
    );
  }
}

export default TableRow;
